packages/core-cairo/src/print.ts

```diff
@@ -6,7 +6,9 @@ import { formatLines, spaceBetween, Lines } from './utils/format-lines';
 import { getSelfArg } from './common-options';
 import { compatibleContractsSemver } from './utils/version';
 
-export function printContract(contract: Contract): string {
+export function printContract(contract: Contract, isAccount?: boolean): string {
```

Comment:
1. I think it makes sense to make the `isAccount` parameter required. Then we'll be sure that we have updated all the calls to it throughout the codebase

2. It's not obvious that the 2nd param of `printContract(contract, true)` tells if the contract is an account or not. Also, we may need other flags/params in the future. What do you think on making it a part of the options dictionary?
```suggestion
export function printContract(contract: Contract, options: { isAccount: boolean }): string {
```
This way calls will looks like this
```
printContract(contract, { isAccount: false })
// or
printContract(contract, { isAccount: opts?.kind === 'Account' })
```